calico/_includes/content/calico-windows-install.md

```diff
@@ -0,0 +1,114 @@
+{%- if include.networkingType == "vxlan" %}
+1. Ensure that BGP is disabled since you're using VXLAN.
+   If you installed Calico using operator, you can do this by:
+
+   ```bash
+   kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
+   ```
+   If you installed Calico using the manifest from https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml then BGP is already disabled.
+{%- else %}
+1. Enable BGP service on the Windows nodes (instead of VXLAN).
```

Comment:
Not sure (instead of VXLAN) clause adds any clarity. Basically, tell the user what to do, not what not to do.